packages/gutenpride-template/README.md

```diff
@@ -0,0 +1,15 @@
+# Gutenpride Template
+
+
+This is a template for @wordpress/create-block that is the finished version of the block in the official [WordPress tutorial](https://github.com/WordPress/gutenberg/tree/master/docs/designers-developers/developers/tutorials/create-block).
```

Comment:
This one is tricky, but if we provide the following URL then it will get optimized in the developer portal and users will stay within it:
```suggestion
This is a template for @wordpress/create-block that is the finished version of the block in the official [WordPress tutorial](/docs/designers-developers/developers/tutorials/create-block).
```

Reply:
done!